examples/sampling/stratified-sampling/StratifiedSampler.cs

```diff
@@ -0,0 +1,89 @@
+// <copyright file="StratifiedSampler.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Trace;
+
+namespace StratifiedSamplingByQueryTypeDemo;
+
+// This is a Proof of Concept of a composite sampler that can used to
+// achieve stratified sampling. It shows how to use one or more span attributes
+// (that need to be available at the start of an activity) to perform stratified sampling.
+// Important considerations:
```

Comment:
please move this to the readme.md file too

Reply:
Sure, will do.